classes/Ajax.php

```diff
@@ -679,6 +679,12 @@ public function create_or_update_annoucement() {
 			$form_data['ID'] = Input::post( 'announcement_id' );
 		}
 
+		if ( ! empty( $form_data['ID'] ) ) {
+			if ( ! tutor_utils()->can_user_manage( 'announcement', $form_data['ID'] ) ) {
+				wp_send_json_error( array( 'message' => __( 'Access Denied', 'tutor' ) ) );
```

Comment:
`wp_send_json_error( array( 'message' => tutor_utils()->error_message() ) );`

**error_message** this method by default take an argument with 401 value. So, no need to pass any argument in this context.